Resources/Prototypes/_Moffstation/Entities/Objects/Fun/toys.yml

```
@@ -0,0 +1,14 @@
+﻿- type: entity
+  parent: CardSword
+  id: CardSwordBorg
+  name: padded cardboard sword
+  description: A cardboard tube fashioned into the shape of a sword - now safe for cyborg use!
+  components:
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 1.25
+    damage:
+      types:
+        Blunt: 0
+    soundHit:
+      path: /Audio/Items/Toys/card_tube_bonk.ogg
```

```suggestion
  - type: MeleeWeapon
    damage:
      types:
        Blunt: 0
```
Should work like this. Top level fields of components are inherited.